SwiftyDropbox.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'SwiftyDropbox'
-  s.version      = '6.0.3b'
+  s.version      = '7.0.0'
```

Comment:
think our internal version should be 7.0.0a since it is different than SwiftyDropbox 7.0.0

Reply:
ah good idea, done